drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/resources/assets/simple-item-def.dmn

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2016 Red Hat, Inc. and/or its affiliates.
```

Comment:
Where is this and scesim-dmn.scesim below used?

Reply:
And this is in main, should probably be in tests. I see there are two of these in the repository.

Reply:
@Rikkola 
Yup, thanks!

Reply:
@Rikkola These dmn files are already available in test/resources folder.After a discussion with @gitgabrio, we decided to delete these and keep the original ones.